src/Transformer/EventVolume.ts

```diff
@@ -0,0 +1,23 @@
+import { v2 } from "@datadog/datadog-api-client";
+
+interface EventVolume {
+    time: string;
+    count: number;
+}
+
+const eventVolume = {
+    convertToMetricPoints: (eventVolumes: EventVolume[]): v2.MetricPoint[] => {
+        const points: v2.MetricPoint[] = [];
+
+        for (const volume of eventVolumes) {
+            points.push({
+                timestamp: new Date(volume.time).getTime() / 1000, // Convert to seconds
+                value: volume.count
+            });
+        }
+
+        return points;
+    }
```

Comment:
Nothing at all wrong with this, just an opportunity to mention the [Array functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) if you're going to be writing more JS 😉 I use them a lot, so this whole function could be

```ts
return eventVolumes.map((volume) => ({
    timestamp: new Date(volume.time).getTime() / 1000, // Convert to seconds
    value: volume.count
}))
```